### PR TITLE
STREAMS-1322 - Disable CORS by defaullt

### PR DESCRIPTION
**Pull request by guillaumelamirand**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links

### CODE REVIEW

**Review by allandenis38 — APPROVED** [review]

**Comment by allandenis38** [comment]
@lbadenhop PR ready for review.

File: content/en/docs/install/_index.md [hunk 1]
```diff
@@ -412,9 +412,11 @@ To disable SSL/TLS (not recommended for production use), see [Helm parameters](#
 
 #### Ingress CORS
 
-Cross-Origin Resource Sharing (CORS) is enabled by default with the default values defined in [Nginx documentation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#enable-cors). You can disable it by setting the [Helm parameters](#helm-parameters) `ingress.annotations.nginx.ingress.kubernetes.io/enable-cors` to `"false"`.
+Cross-Origin Resource Sharing (CORS) is disabled by default. You can enable it by setting the [Helm parameter](#helm-parameters) `ingress.annotations.nginx.ingress.kubernetes.io/enable-cors` to `"true"`. For instance:
```
Comment (lbadenhop) [inline content/en/docs/install/_index.md:415]:
```suggestion
Cross-Origin Resource Sharing (CORS) is disabled by default. You can enable it by setting the [Helm parameter](#helm-parameters) `ingress.annotations.nginx.ingress.kubernetes.io/enable-cors` to `"true"`:
```

File: content/en/docs/install/_index.md [hunk 2]
```diff
@@ -412,9 +412,11 @@ To disable SSL/TLS (not recommended for production use), see [Helm parameters](#
...
 
-Otherwise, you can configure it by adding annotations to the `ingress` parameter (refer to [Nginx documentation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#enable-cors) for further details):
+* Add `--set-string "ingress.annotations.nginx\.ingress\.kubernetes\.io/enable-cors"="true"` in the Helm Chart installation command (mind the `--set-string` here).
```
Comment (lbadenhop) [inline content/en/docs/install/_index.md:417]:
```suggestion
* Add `--set-string "ingress.annotations.nginx\.ingress\.kubernetes\.io/enable-cors"="true"` in the Helm Chart installation command (make sure you enter `--set-string`).
```

**Review by lbadenhop — COMMENTED** [review]
@allandenis38  See suggestions

**Pull request merged**